### PR TITLE
Added a governance tab with a proposal creatation feature.

### PR DESCRIPTION
**Pull request by KolbyML**
This proposal adds a governance tab to pivx's new gui. It also adds a proposal creation feature so you can easily create a proposal without using the console.

This is meant to make the governance process easier for the common user who isn't tech-savvy and should allow more people to participate in the governance process both in voting and in creation.

This is the governance tab it is based off the old one pivx had, but with some modification to better fit the gui and with the addition of a dark theme for it.
image

It also adds a proposal creation feature which allows you to create proposals in a graphical user interface instead of a console opening up governance to more people than ever before and breaking the barrier to entry. 
image

### CODE REVIEW

**Comment by akshaynexus** [comment]
Is there a image of how the dark ui looks for governance's it possible to conform to th new Pivx ui's dialog system?wherin it isn't a popup,se the encrypt wallet dialog to see how its handled on it.

**Comment by KolbyML** [comment]
Yeah sure I will post that here

File: src/qt/dogecash/forms/proposaldialog.ui [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+                <string>Cancel</string>
+              </property>
+            </widget>
+          </item>
+        </layout>
+      </item>
+    </layout>
+  </widget>
+  <resources>
+    <include location="../phore.qrc"/>
```
Comment (akshaynexus) [inline src/qt/dogecash/forms/proposaldialog.ui]: phore qrc include, needs change to dogecash.qrc

**Comment by KolbyML** [comment]
image

File: src/qt/dogecash/governancepage.cpp [hunk 2]
```diff
@@ -142,7 +162,21 @@ void GovernancePage::setExtendedProposal(CBudgetProposal* proposal)
         updateProposalList();
 }
 
-void GovernancePage::on_UpdateButton_clicked()
+void GovernancePage::onUpdateButtonclicked()
 {
+    LogPrintf("55\n");
```
Comment (akshaynexus) [inline src/qt/dogecash/governancepage.cpp]: Not required logprintf ,and some log printfs remaining form I guess testing

**Comment by KolbyML** [comment]
Good catch I will fix that black text up also I forgot to check the dark theme for the creation proposal

File: src/qt/dogecash/governancepage.cpp (hunk 2 above)
Comment (KolbyML) [reply src/qt/dogecash/governancepage.cpp]: Yep xd I will remove that

**Review by Liquid369 — COMMENTED** [review]
Looks good
uTACK

Will test in AM

**Comment by KolbyML** [comment]
Ok dark theme is now fixed for the tab also now there are also some improvements for light theme as well.
image
image

**Comment by KolbyML** [comment]
Ok so I just squashed the new commits I made

**Comment by akshaynexus** [comment]
Looks good,wil test it shortly

**Comment by akshaynexus** [comment]
Small change needed before merge,use the svg isntead of png for the governance active and other icons needed for governance ui

**Comment by KolbyML** [comment]
You guys don't have any svg's for governance active in the image folder so if you have a copy i can add or something.

**Comment by akshaynexus** [comment]
Sure,ill link it in a min

**Comment by akshaynexus** [comment]
https://github.com/dogecash/dogecash-explorer/blob/master/public/img/governance.svg

https://github.com/dogecash/dogecash-explorer/blob/master/public/img/governance_white.svg

**Comment by KolbyML** [comment]
I will do that when I drive home today

**Comment by akshaynexus** [comment]
cool

**Comment by KolbyML** [comment]
Ok I added the svg's and did a squash

**Pull request merged**